src/main/java/net/dv8tion/jda/core/entities/impl/GuildImpl.java

```diff
@@ -363,6 +363,53 @@ protected void handleResponse(Response response, Request<List<Ban>> request)
         };
     }
 
+    @Nonnull
+    @Override
+    public RestAction<Ban> getBanById(long userId)
+    {
+        return getBanById(Long.toUnsignedString(userId));
+    }
+
+    @Nonnull
+    @Override
+    public RestAction<Ban> getBanById(String userId)
+    {
+        if (!isAvailable())
+            throw new GuildUnavailableException();
```

Comment:
There is no need to check this for a REST request, we removed all of these in the experimental branch

[Commit](https://github.com/DV8FromTheWorld/JDA/commit/3a580d0ab521d24f5b6cf462ed9d9ac51a1a0206)